bouncer/authentication/models.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.utils.crypto import get_random_string
+
+
+class AuthToken(models.Model):
+    key = models.CharField(_("Key"), max_length=32, primary_key=True, editable=False)
```

Comment:
Please remove `_("Key")`, as we don't have translated strings anywhere else. If we decide to use it, we'll do it later on.